data/src/main/java/org/cryptomator/data/cloud/pcloud/RootPCloudFolder.java

```diff
@@ -0,0 +1,27 @@
+package org.cryptomator.data.cloud.pcloud;
+
+import com.pcloud.sdk.RemoteFolder;
+
+import org.cryptomator.domain.Cloud;
+import org.cryptomator.domain.PCloud;
+
+class RootPCloudFolder extends PCloudFolder {
+
+	private final PCloud cloud;
+	private static final long rootFolderId = RemoteFolder.ROOT_FOLDER_ID;
+
+	public RootPCloudFolder(PCloud cloud) {
+		super(null, "", "", rootFolderId);
```

Comment:
`RemoteFolder.ROOT_FOLDER_ID` could be an inline variable or?
```suggestion
		super(null, "", "", RemoteFolder.ROOT_FOLDER_ID);
```

Reply:
> 
> 
> `RemoteFolder.ROOT_FOLDER_ID` could be an inline variable or?

Totally. Changed it. Have to cast it to `long` though.

Reply:
> Have to cast it to long though.

int to long casts are implicit and safe. No need to redeclare the constant, but doesn't hurt either. Please stick to all uppercase naming convention in this case, though.